### PR TITLE
fix unit tests when using windows

### PR DESCRIPTION
**Pull request by Remo**
It's not complete, still some broken tests. Mostly related to console and processes, one example below. I'll try to continue when I'm home.

```
2) PhpBench\Tests\System\RunTest::testCommandWithReportConfigurationUnknown
Failed asserting that '

  [Symfony\Component\Console\Exception\RuntimeException]
  Too many arguments.
```

### CODE REVIEW

File: bin/phpbench.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@echo off
+php.exe "%~dp0\phpbench.php" %*
```
Comment (dantleech) [inline bin/phpbench.bat]: Is this file necessary? I notice that well established libraries such as phpspec do not have this file. https://github.com/phpspec/phpspec/tree/master/bin

File: tests/System/SystemTestCase.php [hunk 2]
```diff
@@ -49,6 +49,7 @@ public function phpbench($command, $workingDir = '.')
     {
         chdir($this->getWorkingDir($workingDir));
         $bin = __DIR__ . '/../../bin/phpbench --verbose';
+        $bin = str_replace('/', DIRECTORY_SEPARATOR, $bin);
```
Comment (dantleech) [inline tests/System/SystemTestCase.php]: Hmm, I thought that `/` worked in Windows these days, there are many instances of this. Which version of PHP are you using?

File: bin/phpbench.bat (hunk 1 above)
Comment (Remo) [inline bin/phpbench.bat]: Kind of. I can do this:
```
C:\Work\phpbench>bin\phpbench
phpbench version 0.10.0-dev
```

But when starting a process using `new Process` it doesn't work. Removing that bat file changes this:
`Tests: 245, Assertions: 461, Errors: 17, Failures: 31.`
to this:
`Tests: 245, Assertions: 504, Errors: 5, Failures: 15.`

File: tests/System/SystemTestCase.php (hunk 2 above)
Comment (Remo) [inline tests/System/SystemTestCase.php]:
```
C:\Work\phpbench>php -v
PHP 5.6.12 (cli) (built: Aug  6 2015 12:06:17)
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies
    with Xdebug v2.3.3, Copyright (c) 2002-2015, by Derick Rethans
```

Never heard that you can use `/` in the console. I'm on Windows 10:
```
C:\Work\phpbench>bin/phpbench
'bin' is not recognized as an internal or external command,
operable program or batch file.

C:\Work\phpbench>bin\phpbench
phpbench version 0.10.0-dev
...
```

Comment (Remo) [inline tests/System/SystemTestCase.php]: I just fixed this one occurence because I wanted to know how/if you'd want to take care of it. I've seen a few more instances. Most unit tests are working though, it's not that bad.

File: bin/phpbench.bat (hunk 1 above)
Comment (dantleech) [inline bin/phpbench.bat]: Strange, I really need to get a Windows VM setup locally :)

Comment (Remo) [inline bin/phpbench.bat]: Sorry, the above is wrong. This doesn't work `C:\Work\phpbench>bin\phpbench`.
Too many open projects /-:
It works because composer creates a bat file when installing it as a library. https://getcomposer.org/doc/articles/vendor-binaries.md#what-about-windows-and-bat-files-

However, when I run `composer update` it doesn't seem to create that file.

File: tests/System/SystemTestCase.php (hunk 2 above)
Comment (Remo) [inline tests/System/SystemTestCase.php]: You probably have something slightly different in mind. Functions like `fopen` can handle `/`, but here we are starting a process, nothing related to PHP, only about processes / parameter.

Comment (dantleech) [inline tests/System/SystemTestCase.php]: Ah ok, yes your right. I would probably go for `implode(DIRECTORY_SEPARATOR, array('path', 'to', 'something'))` in that case.

File: bin/phpbench.bat (hunk 1 above)
Comment (dantleech) [inline bin/phpbench.bat]: Maybe try removing the vendor directory, removing composer.lock and
doing a `composer install`, perhaps ?

Comment (Remo) [inline bin/phpbench.bat]: I'm afraid, no luck.

Comment (WouterJ) [inline bin/phpbench.bat]: Just use `php bin/phpbench` and things work perfectly :)

Comment (Remo) [inline bin/phpbench.bat]: @WouterJ that only works if you use it as a library, composer doesn't create those for standalone projects
